hack/release.sh

```diff
@@ -45,17 +39,31 @@ function build_release() {
   else
     echo "Untagged release, will NOT update release labels"
     LABEL_YAML_CMD=(cat)
+    TAG="latest"
   fi
 
+  readonly UPGRADE_JOB="upgrade-to-${TAG}.yaml"
```

Comment:
Do you also need to update the config/upgrade/v0.14/README.md to point to the newly renamed file?

Reply:
and than I think I find it strange that the `0.14` than would have a `TAG` that is different than `0.14` (e.g. `latest` or `0.15`)

Reply:
I will keep the folder of config/upgrade/v0.14.0 unchanged. If we need script for upgrade in future version, we can change the release.sh later.